src/series8/UserTemperature.java

```diff
@@ -0,0 +1,24 @@
+package series8;
+
+import java.util.Scanner;
+
+public class UserTemperature {
+	public static void main(String[] args)
+	{
+		int user_temperature;
+		
+		Scanner sc = new Scanner(System.in);
+		
+		user_temperature = sc.nextInt();
+		
+		if(user_temperature >= 40 && user_temperature < 80)
```

Comment:
Let me add a condition and you update the problem accordingly: So even the temperature 40 and 80 are supposed to be abnormal.